packages/aws-cdk/lib/api/resource-import/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './importer';
+export * from './migrator';
```

Comment:
sort of a meta question -- is there a way to block us from importing directly from the file itself when we have an `index.ts`?

i want to disawllow `import * as blah from 'aws-cdk/lib/api/resource-inport/importer';` in favor of `import * as blah from 'aws-cdk/lib/api/resource-import';` in this case, so that we don't have multiple ways of doing the same thing

Reply:
You could write an eslint rule for it, might need a plugin. There can be good reasons to import directly though. My current concern is that anything _outside_ of `api` should only import form the barrel files. Within `api` I don't mind so much.